### PR TITLE
feat: Display no available parameters

### PR DESCRIPTION
**Pull request by dwesolow**
## 📝 Description

image

## ✨ Type of Change

Select the type of change your PR introduces:

- [X] 🚀 **New feature** – Non-breaking change which adds functionality

## 🧪 Testing Scenarios

Describe how the changes were tested and how reviewers can test them too:

- [X] ✅ Tested manually
- [ ] 🤖 Run automated end-to-end tests

## ✅ Checklist

Before submitting the PR, ensure the following:

- [X] 🔍 PR title is clear and meaningful
- [ ] ✍️ PR description clearly explains the changes and their reason
- [ ] 📝 I have linked the PR to the corresponding GitHub Issues, if any
- [ ] 💬 I have commented my code, especially in hard-to-understand areas
- [ ] 📄 I have made corresponding changes to the documentation
- [X] ✅ I have added tests that prove my fix is effective or my feature works

### CODE REVIEW

**Review by romanowska — DISMISSED** [review]

**Review by jpggvilaca — DISMISSED** [review]
LGTM, but let's add a test for this later

File: web_ui/src/pages/project-details/components/project-model/model-configurable-parameters/trained-model-configuration-parameters/trained-model-configuration-parameters.component.tsx [hunk 1]
```diff
@@ -105,5 +105,14 @@ export const TrainedModelConfigurationParameters = ({ taskId }: TrainedModelConf
         );
     }
 
+    if (isEmpty(data.training) && isEmpty(data.datasetPreparation.augmentation)) {
+        return (
+            <NotFound
+                heading={'No training parameters'}
+                content={<Text>The model does not have any training parameters</Text>}
```
Comment (MarkRedeman) [inline web_ui/src/pages/project-details/components/project-model/model-configurable-parameters/trained-model-configuration-parameters/trained-model-configuration-parameters.component.tsx]: I think it would be good to emphasize it not having any **configurable** training parameters.

```suggestion
                content={<Text>The model does not have any configurable training parameters</Text>}
```

Later I think we should extend this page to also show the non-configurable parameters. I.e. all models always have an input size, a batch size, learning rate etc which we can show here.

Comment (dwesolow) [reply web_ui/src/pages/project-details/components/project-model/model-configurable-parameters/trained-model-configuration-parameters/trained-model-configuration-parameters.component.tsx]: Updated

**Review by jpggvilaca — APPROVED** [review]

File: web_ui/src/pages/project-details/components/project-model/model-configurable-parameters/trained-model-configuration-parameters/trained-model-configuration-parameters.component.tsx (hunk 1 above)
Comment (dwesolow) [reply web_ui/src/pages/project-details/components/project-model/model-configurable-parameters/trained-model-configuration-parameters/trained-model-configuration-parameters.component.tsx]: Upated

**Review by pplaskie — APPROVED** [review]
